### ISSUE TITLE
Description comments are duplicated/repeated twice for all users when viewed on Dashboard

### ISSUE THREAD

**Issue by calisadora** [thomaschampagne/elevate#712]
### Step 1: Are you in the right place?

  * Your problem may have been already covered in the FAQ. Did you check https://github.com/thomaschampagne/elevate/wiki/Frequently-Asked-Questions?
  * Your problem or feature request may already exists in project issues section. Did you check https://github.com/thomaschampagne/elevate/issues?

### Step 2: Describe your environment

  * Plugin version: _____
  * Chrome/Opera/Chromium version: Version 70.0.3538.110 (Official Build) (64-bit)_____
  * OS version: ____10.14.2 (18C54)_
  
### Step 3: Describe the problem:
Comments that are entered in the Description field appear twice when viewed in the dashboard.  This occurs for every user visible on the dashboar.
#### Steps to reproduce:

  1. __Open Strava___
  2. __Select Dashboard___
  3. ___See comments from Description field twice__
  
#### Actual Behavior:

  * What happened?
  * Do you have javascript errors logs in browser console?

  > Press F12 to see developer console and javascript logs

  ```javascript
  Copy and paste your console logs here.
  ```
  
#### Expected Behavior:

  * What did you expect to happen?
  Expect to see comments that were written once to appear only once
#### Media attachement:

>You can upload/attach any images to this ticket.

#### Providing your Elevate synced athlete activities can be usefull to solve the problem

> To export your activities click on "sync button" (top right and corner), then "Backup activities". You can zip the file and upload it here.

**Comment by thomaschampagne** [comment]
Strange if you disable extension what happen? Do you have a screenshot :) ?

**Comment by calisadora** [comment]
when I remove the Stravini exception, the duplicates go away, but so does
the desired screen layout.  Attached shows the dups

Thanks!

**Comment by calisadora** [comment]
also, the first/top display of the comment is not linked.  the 2nd display
is hyperlinked to the ride details for that ride

**Comment by thomaschampagne** [comment]
@calisadora i can't see images from the ticket directly. Problem is solved, right? If yes you can close the ticket :)

**Comment by calisadora** [comment]
well, problem is solved only if I remove the Stravini extension.  When the extension is active, then I see the duplicates.   So, I wouldn't say that the problem is solved.  

<img width="865" alt="screen shot 2018-12-18 at 4 37 54 pm" src="https://user-images.githubusercontent.com/45980153/50190976-83a12880-02f9-11e9-8310-95a72e77f7ce.png">

**Comment by dlh3** [comment]
@calisadora the problem you are reporting is with [Stravini](https://chrome.google.com/webstore/detail/stravini/ilogdlddgndehmpabkafohhfapaopgkd).  That plugin is different from this extension, [Elevate](https://chrome.google.com/webstore/detail/elevate-for-strava/dhiaggccakkgdfcadnklkbljcgicpckn) (formerly Stravistix).

I disabled the Elevate plugin and installed Stravini, and witnessed the behaviour you reported.  Since I had Elevate disabled, I've ruled out this plugin causing the issue.  Closing this issue as invalid.
